generator/internal/cc_stub_generator.cc

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Google Inc.  All rights reserved
```

Comment:
Filename: stub_cc_generator? So that we have two classes {client, stub} and two file types {cc, header} and the structure is {class}_{file}_generator.{cc, h}

Reply:
Doh! Don't know what I was thinking. That's a much better name.